packages/adminrouter/extra/src/includes/server/master.conf

```diff
@@ -158,7 +158,9 @@ location ~ ^/service/(?<service_path>.+) {
 
     include includes/proxy-headers.conf;
     include includes/websockets.conf;
-    include includes/disable-request-response-buffering.conf;
+
+    #Disable buffering to allow real-time protocols
```

Comment:
Comment amended. 

The previous comment was taken from the original `nginx.conf` file.